bin/alluxio-mount.sh

```diff
@@ -139,6 +148,7 @@ function mac_hfs_provision_sectors() {
 function mount_ramfs_linux() {
   init_env $1
 
+  ALLUXIO_WORKER_MEMORY_SIZE=$(${BIN}/alluxio getConf alluxio.worker.memory.size)
```

Comment:
also should the reference to ALLUXIO_WORKER_MEMORY_SIZE below (line 161) be updated to point to the a site property